### PR TITLE
Release of 2.0.0 (@next)

### PR DESCRIPTION
**Pull request by tdeekens**
This pull request aggregates everything for the 2.0.0 release.

- Remove `expand`
- Add `renderProp`
- Improved docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,117 +130,67 @@ This is way more readable and easier to follow along. At the same time everythin
...
-
-  it('should pass `onClick` to `<button>`', () => {
-    nestedWrapper.find('button').prop('onClick')();
-
-    expect(nestedWrapperProps.onChange).toHaveBeenCalled();
-  });
-});
+// this is the actual integration
+ShallowWrapper.prototype.renderProp = renderProp;
+ShallowWrapper.prototype.until = until;
```
Comment (dferber90) [inline README.md]: @tdeekens We should mention in the release notes that the users need to remove the `expand` integration when upgrading.

Comment (dferber90) [inline README.md]: I was thinking whether it's easier to export a `configure` function, which would be used like this:

```js
import configure from `@commercetools/enzyme-extensions`

configure(enzyme)
// or
configure(ShallowWrapper)
```

Comment (tdeekens) [reply README.md]: Yep. I wanted to write them as a GitHub Release.

Comment (tdeekens) [reply README.md]: That's a nice idea and should work. I would like it only working with passing in `enzyme` as people will have that imported in their configuration files already to setup a react-adpater but I don't think it'll work.

Comment (tdeekens) [reply README.md]: 2d5d7ef

**Pull request merged**